examples/all-clusters-app/all-clusters-common/gen/attribute-size.cpp

```diff
@@ -195,14 +195,14 @@ uint16_t emberAfCopyList(ClusterId clusterId, EmberAfAttributeMetadata * am, boo
             // Struct _DeviceType
             _DeviceType * entry = reinterpret_cast<_DeviceType *>(write ? src : dest);
             copyListMember(write ? dest : (uint8_t *) &entry->type, write ? (uint8_t *) &entry->type : src, write, &entryOffset,
-                           sizeof(entry->type)); // DEVICE_TYPE_ID
+                           sizeof(entry->type)); // DEVTYPE_ID
             copyListMember(write ? dest : (uint8_t *) &entry->revision, write ? (uint8_t *) &entry->revision : src, write,
                            &entryOffset, sizeof(entry->revision)); // INT16U
             break;
         }
         case 0x0001: // server list
         {
-            entryLength = 2;
+            entryLength = 4;
```

Comment:
This is autogenerated code. I agree it would be nice if we could use `sizeof` but since attributes can also stored `struct` members that won't work :(

Reply:
too bad - or are the structs perhaps defined with `__attribute__((packed))` ? :)